buildlib/docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
     image: centos7-mofed5.0-cuda10.1
     build:
       context: .
+      network: host
```

Comment:
why needed in this PR?

Reply:
It's a part of the related verification.

Reply:
what is it doing ? would AZP fail without this line, now that we removed deps?

Reply:
It isn't used in AZP, just to build UCX build/test docker environment manually with docker-compose as described here https://github.com/openucx/ucx/blob/master/buildlib/azure-pipelines.md#release-images
I'll put it in a separate PR.